src/features/currentTodo.ts

```diff
@@ -1,10 +1,19 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSelector, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { RootState } from '../app/store';
 
 const initialState = null as Todo | null;
 
 export const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: { set: (_, { payload }) => payload },
 });
+
+const currentTodo = (state: RootState) => state.currentTodo;
```

Comment:
Consider explicitly typing the `currentTodo` selector function to ensure type safety and prevent potential TypeScript errors. This can be done by specifying the return type as `Todo | null`.